### ISSUE TITLE
[Compile Error] Does not compile on Linux

### ISSUE THREAD

**Issue by MarcoKull** [masterfeizz/DaedalusX64-3DS#55]
I compiled the CTR_RELEASE under Linux and it will stop compiling with the following error:

```
Scanning dependencies of target N3DS
[ 99%] Generating DaedalusX64.3dsx
Cannot open dir

make[2]: *** [CMakeFiles/N3DS.dir/build.make:82: DaedalusX64.3dsx] Error 1
make[1]: *** [CMakeFiles/Makefile2:100: CMakeFiles/N3DS.dir/all] Error 2
make: *** [Makefile:104: all] Error 2
```

However, changing line 367 in `Source/CMakeLists.txt` from
`set(APP_ROMFS_DIR "${PROJECT_SOURCE_DIR}/SysCTR/Resources/RomFS")`
to
`set(APP_ROMFS_DIR "${PROJECT_SOURCE_DIR}/SysCTR/Resources/romfs")`
resolves the issue. Seems like this is because Linux is case sensitive.

**Comment by BlowZed** [comment]
interesting...

**Comment by pablomancera** [comment]
Yeah, I had the same error on my machine, it was one file with first letter uppercase, and referenced as all lowercase, find it and rename it

**Comment by MisterPrintf** [comment]
Issue here is that Linux is case-sensitive.

**Comment by Idiot64** [comment]
why is that an issue, like you think thats bad?

sorry if i misunderstood, but case sensitive for file names isn't crap

**Comment by MisterPrintf** [comment]
> 
> 
> why is that an issue, like you think thats bad?
> 
> sorry if i misunderstood, but case sensitive for file names isn't crap

Not an issue with Linux here; I am also in support of case-sensitivity. The issue is with the build script.

**Comment by tomrow** [comment]
> why is that an issue, like you think thats bad?
> 
> sorry if i misunderstood, but case sensitive for file names isn't crap

TLDR, Linux and Unix-like systems don't consider uppercase and lowercase letters the same, and a critical folder can't be found because of it.
